README.md

```diff
@@ -98,29 +98,30 @@ docker run -it --rm ^
 api3/airkeeper:latest remove --stage dev --region us-east-1
 ```
 
-## Dev instructions
+## Development instructions
 
-Make sure to have the following dependencies installed:
+Make sure to have yarn installed, then run:
 
-- npm
-- serverless framework (https://www.serverless.com/framework/docs/getting-started)
+```sh
+yarn install
+```
 
 ### Running Airkeeper locally
 
-In order to run Airkeeper locally, you will need to follow these steps:
+Follow these steps to run Airkeeper locally:
 
 1. Open a new terminal and start a new local ethereum node.
 1. Deploy all required contracts (RrpBeaconServer, DapiServer, etc) and set everything up (whitelisting, sponsorship,
    etc).
 1. Switch to the Airkeeper root directory and run `yarn install`.
-1. Add proper values to the `config.json` and `airkeeper.env` files.
-1. Finally run `yarn sls:invoke-local:psp` to invoke the `psp.beaconUpdate` handler function or run
+1. Add appropriate values to the `config.json` and `airkeeper.env` files.
```

Comment:
```suggestion
1. Add appropriate values to the `config.json` and `airkeeper.env` files.
```
Shouldn't this be `airkeeper.json` :detective:

Reply:
It is, nicely spotted, I forgot about this PR. Thanks for checking!